### PR TITLE
docs(docs/debug.md): debug documentation

### PR DESCRIPTION
**Pull request by SuelenKarbivnychyy**
created a documentation file to sucessfully set up a vs code debugger to debug this project

#74

### CODE REVIEW

**Comment by SuelenKarbivnychyy** [comment]
@sansyrox please review.

File: docs/css_file.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+# Introduction
+
+This document reflect the accomplished tasks during the MLH 12 weeks Summer Fellowship.
```
Comment (sansyrox) [inline docs/css_file.md:3]:
```suggestion
This document reflects the accomplished tasks during the 12 weeks of the MLH Summer Fellowship.
```

File: docs/css_file.md
Comment (sansyrox) [inline docs/css_file.md:1]: Hey @SuelenKarbivnychyy 👋 

We shouldn't be documenting what a developer did, rather, we should be documenting *about* the work done by the developer.

We shouldn't using statements like - `This document reflect the accomplished tasks during the MLH 12 weeks Summer Fellowship.`
or 
`These issues were closed by [Suelen Karbivnychhy](https://github.com/SuelenKarbivnychyy)`  or use a particular issue number.

You can have a look here if you need an example of the documentation - https://robyn-docs.vercel.app/documentation

Do let me know if you still have any questions. 😄

File: docs/debug.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
+# This document is intended for developers who want to setup a debugger for this project in VS Code.
```
Comment (sansyrox) [inline docs/debug.md:1]: This heading is too long. This should be an opening statement imo. We should have a simpler heading like `## DEBUGGING`

File: docs/debug.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+## ⚙️ Follow the steps:
```
Comment (sansyrox) [inline docs/debug.md:3]:
```suggestion
## ⚙️ Follow the steps:
```

Comment (sansyrox) [inline docs/debug.md:3]: Nicee

File: docs/debug.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+            "--build"
+            ],
+            "stopOnEntry": false,
+            "console": "integratedTerminal",
+        }
+    ]
+}
+```
+7. In VSCode menu test with:
+    - Click on `Run` -> `Start Without Debuggig` "or press `Ctrl + F5`" - check that the output has no errors
```
[the remark this answers is not in the document]
Comment (SuelenKarbivnychyy) [reply docs/debug.md:65]: > `Start without debugging`? I am not using `VSCode` but shouldn't we do `Start with debugging`?

@sansyrox 
We do `Start without debugging` to simple run the application without needed to "update and run the script `<./build.sh>`", and to see if there are no error.
We do `Start with debugging` when we want to use break points, so we can evaluate line by line. 
Who wants to use the `Vs code debugger` understands how does that process works.

File: docs/debug.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+            "console": "integratedTerminal",
+        }
+    ]
+}
+```
+7. In VSCode menu test with:
+    - Click on `Run` -> `Run Without Debugging` "or press `Ctrl + F5`" - to check that the output has no errors
+
+8. To debugg it, In VSCode menu:
+    - Add break point on the lines you want to debbug
```
Comment (sansyrox) [inline docs/debug.md:68]:
```suggestion
    - Add break point on the lines you want to debug
```

File: docs/debug.md [hunk 6]
```diff
@@ -0,0 +1,69 @@
+# Debugger Setup
+
+### Introduction
+
+This document is intended for developers who want to setup a debugger for this project in VS Code.
```
Comment (sansyrox) [inline docs/debug.md:5]:
```suggestion
This document is intended for developers who want to setup a debugger for Starfyre in VS Code.
```

**Review by sansyrox — APPROVED** [review]
LGTM :smile:

**Pull request merged**